subsys/fw_info/Kconfig

```diff
@@ -82,7 +82,7 @@ config FW_INFO_HARDWARE_ID
 config FW_INFO_VERSION
 	int "Version number of the fw_*_info structs (8 bits)."
 	range 0 255
-	default 1
+	default 2
```

Comment:
On a side note, should this field even be visible for the users? IE should it have a prompt? IMO this could be considered a hidden option.

Reply:
Done